scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -1711,10 +1664,8 @@ cdef class NumericalInverseHermite(Method):
                   u_resolution=1e-12,
                   tol=None,
                   construction_points=None,
-                  max_intervals=100000,
```

Comment:
We can't remove this from `NumericalInverseHermite` for backward compatibility reasons.

Reply:
Oh, you are right, this parameter was already there in the old version. A doc deprecation should work though?

Reply:
> A doc deprecation should work though?

I don't think we should deprecate it either: it will unnecessarily increase maintenance load later. I think it's safe to leave this as-is.

Reply:
I thought for consistency with TDR and PINV, it is better to remove it. Increasing the number of iterations is probably a very rare case (especially once we change the default to 1e6 after the deprecation). At some point, we need to work a bit on this function anyways (tol deprecation), so I would not mind the extra work to remove `max_intervals`.

Reply:
Oh, I see your point. Let's deprecate `max_intervals` then.

Reply:
cool, then I will keep it as it is